Device/Device/doc/spec.md

```diff
@@ -153,6 +153,10 @@ Obviously, in order to toggle the referred switch, this attribute value will hav
     + Attribute type: [DateTime](https://schema.org/DateTime)
     + Optional    
 
++ `owner` : Entity's owners: an array of entities that represents the owners of a Device.
+    + Attribute type: List of [Person]( http://schema.org/Person) or [Organization](https://schema.org/Organization).
```

Comment:
list of references to, because we don't expect to inline them, do we?

Reply:
(y)